doc/lua_api.txt

```diff
@@ -5064,6 +5064,9 @@ This is basically a reference to a C++ `ServerActiveObject`
 * `set_acceleration(acc)`
     * `acc` is a vector
 * `get_acceleration()`: returns the acceleration, a vector
+* `set_rotation(rot)`
+    * `rot` is a vector (radians)
```

Comment:
Maybe add that y component is yaw, x is pitch, z is (a non-intuitive?) roll?

Reply:
Hmm, not so sure if that will work or explain it well but I can't think of any other way than saying x is x-axis of the object, y is y-axis, and z is z-axis.

Reply:
The way I think of it is the rotation if you put a pin through the axis of the rotation. So Y/Yaw is rotating around a pin through Y.

Not sure how best to put this in words

Reply:
I think that we figured out that, that is not how this works. We all thought it did in my previous PR but it turns out it is different. I think starting [here](https://github.com/minetest/minetest/pull/7331#issuecomment-392302499) should explain what is happening. However, let's not get caught up in this again. The idea is that there should be a way to set the rotation to zero and any other rotation based on raw angles given by lua for each axis, this is the idea around this PR and to add the needed network modifications. Like I said in a couple other comments, I will start another PR that should work the way you think this one does, but that can't be done until this PR is merged. 

I hope that makes sense.

Reply:
Nevermind these docs for now. Indeed in #7331 we discovered what is happening and rotation components do not actually correspond to rotation around the x, y, z axes. Seems fine to leave docs non-specific for now at least.